src/pygeoprocessing/geoprocessing.py

```diff
@@ -3336,8 +3340,8 @@ def _convolve_2d_worker(
         work_queue, write_queue):
     """Worker function to be used by ``convolve_2d``.
 
-    Parameters:
-        Parameters:
+    Args:
+        Args:
```

Comment:
Duplicate `Args:` line here.

Reply:
removed in f7b456ca6eb90cb3da11894808e344f548d2ef2d